### ISSUE TITLE
Mobile menu dropdown enhancements

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_microsites_base#84]
## Submenu on hover

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

**When we show the submenu on hover we also need a way to dismiss it. I'd suggest that we use the ESC key** 

There is also some odd behaviour when using a mouse:

the expose menu button doesn't really make sense in this context (other than as a visual indicator that it's got a submenu). 

**If I click on the submenu arrow, should the submenu remain open no matter where I point my pointer?** 

If an item has two children (second level) and the first of those have children (third level), then I can't access the last second level child. I think this is because the thrid level disappear and then the second moves our of the target area. 

**I think it would be better to expose the third (and lower) levels when hovering on the arrow, rather than the item.**

**Comment by msayoung** [comment]
THis should be fixed by: https://github.com/localgovdrupal/localgov_microsites_base/issues/193

**Comment by markconroy** [comment]
PR https://github.com/localgovdrupal/localgov_microsites_base/pull/268
